pkg/deploy/assets/rp-oic.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "rpServicePrincipalId": {
+            "type": "string"
+        }
+    },
+    "resources": [
+        {
+            "sku": {
+                "name": "Standard_LRS"
+            },
+            "kind": "StorageV2",
+            "properties": {
+                "accessTier": "Hot",
+                "supportsHttpsTrafficOnly": true,
+                "allowBlobPublicAccess": true,
```

Comment:
Should this be set to False?

Reply:
My assumption is that in prod, yes (because we'll have Azure Front Door and won't need blobs publicly accessible), but in dev we wont have Azure Front Door so we need the public access. Is that right?

Reply:
In my read of the documentation - this setting allows anonymous/unauthenticated access to the SA: https://learn.microsoft.com/en-us/azure/storage/blobs/anonymous-read-access-configure?tabs=portal#set-the-storage-accounts-allowblobpublicaccess-property

If unauthenticated access is what we need, this is the right setting, I just want to be sure that is what we need?

Reply:
Caden and I talked offline, he pointed me to https://github.com/openshift/cloud-credential-operator/blob/master/docs/azure_workload_identity.md - the blob only holds public keys, LGTM.